sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -266,6 +266,17 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("SPARK-32374: disallow setting properties for CREATE TEMPORARY VIEW") {
+    withTempView("myabcdview") {
+      val e = intercept[AnalysisException] {
+        sql("CREATE TEMPORARY VIEW myabcdview TBLPROPERTIES ('a' = 'b') AS SELECT * FROM jt")
+      }
+      assert(e.message.contains(
+        "Operation not allowed: CREATE TEMPORARY VIEW ... " +
+          "TBLPROPERTIES (property_name = property_value, ...)"))
```

Comment:
how about: `Operation not allowed: TBLPROPERTIES can't coexist with CREATE TEMPORARY VIEW`